apps/epic-react/src/components/app/navigation/index.tsx

```diff
@@ -76,84 +71,41 @@ const Navigation: React.FC<NavigationProps> = ({navChildren}) => {
                 router.push('/?buy')
               }}
               className="flex cursor-pointer rounded-lg bg-blue-500 px-3 py-2 text-white transition-all  duration-150 ease-in-out hover:bg-blue-600"
+              data-navi-item=""
             >
               Buy Epic React
             </button>
           ) : null}
-          <Link
-            href="/articles"
-            className={twMerge(
-              cx(
-                'rounded-md px-3 py-2 text-text transition-opacity duration-150 ease-in-out hover:opacity-100 sm:opacity-75',
-                {'bg-er-gray-100 sm:opacity-100': location === '/articles'},
-              ),
-            )}
-          >
-            Articles
-          </Link>
-          <Link
-            href="/livestreams"
-            className={twMerge(
-              cx(
-                'rounded-md px-3 py-2 text-text transition-opacity duration-150 ease-in-out hover:opacity-100 sm:opacity-75',
-                {'bg-er-gray-100 sm:opacity-100': location === '/livestreams'},
-              ),
-            )}
-          >
-            Livestreams
-          </Link>
-          <Link
-            href="/podcast/kents-career-path-through-web-development"
-            className={twMerge(
-              cx(
-                'rounded-md px-3 py-2 text-text transition-opacity duration-150 ease-in-out hover:opacity-100 sm:opacity-75',
-                {
-                  'bg-er-gray-100 sm:opacity-100':
-                    location.startsWith('/podcast/'),
-                },
-              ),
-            )}
-          >
-            Podcast
-          </Link>
-          <Link
-            href="/faq"
-            className={twMerge(
-              cx(
-                'rounded-md px-3 py-2 text-text transition-opacity duration-150 ease-in-out hover:opacity-100 sm:opacity-75',
-                {'bg-er-gray-100 sm:opacity-100': location === '/faq'},
-              ),
-            )}
-          >
-            FAQ
-          </Link>
+          {menuItems.map((item) => {
+            return (
+              <Link
+                key={item.label}
+                href={item.slug}
+                className={twMerge(
+                  cx(
+                    'rounded-md px-3 py-2 text-text transition-opacity duration-150 ease-in-out hover:opacity-100 sm:opacity-75',
+                    {
+                      'bg-er-gray-200 sm:opacity-100': location.startsWith(
+                        item.slug,
+                      ),
+                    },
+                  ),
+                )}
+                data-navi-item=""
+              >
+                {item.label}
+              </Link>
+            )
+          })}
           {sellingLive && !isAuthenticated ? (
             <Link
               href="/login"
               className="rounded-lg px-3 py-2 text-text transition-all duration-150 ease-in-out lg:bg-blue-500 lg:text-white lg:hover:bg-blue-600"
+              data-navi-item=""
```

Comment:
what's this attribute do?

Reply:
Ah I see

https://github.com/skillrecordings/products/pull/1534/files#r1605615240